README.md

```diff
@@ -44,11 +48,13 @@ Full demo video: https://www.youtube.com/watch?v=2cKSsBxrvR8
 
 `$ lumber [command]`
 
-- `generate`      generate your admin interface based on your database schema
+- `generate`      generate your back office based on your database schema
 - `update`        update your models's definition according to your database schema
+- `deploy`        deploy your back office application to production
 - `user`          show your current logged user
 - `login`         sign in to your Forest account
```

Comment:
I would say _`Log in to your Forest account`_ than sign in. Most of the people do not understand the difference between sign in and sign up. Just an advise

Reply:
Will be on another PR with a complete review on the help + readme